app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,23 @@
+class PasswordResetsController < ApplicationController
+  def create
+    user = User.find_by_token(params[:token])
+    if user
+      user.password = params[:password]
+      user.generate_token
+      user.save
```

Comment:
Instead of saving you can just use `.update_columns` with above updated attributes.